reference/release-notes/1.26.0.md

```diff
@@ -70,7 +70,18 @@ The following issues exist in the 1.26.0 release and we are working to fix them
 
 * A `program was killed: context canceled` error occurs during appliance initialization even though all services are active. ([LP 2110321](https://bugs.launchpad.net/anbox-cloud/+bug/2110321)).
 * Security patch level for AOSP 15 and AAOS 15 remains unchanged at `2025-05-05` despite applying the latest security patches.([LP 2110323](https://bugs.launchpad.net/anbox-cloud/+bug/2110323)).
-* Unable to stream an instance after deploying the regular Anbox Cloud using Juju([LP 2110194](https://bugs.launchpad.net/anbox-cloud/+bug/2110194)). This issues exists since the 1.25.0 release.
+* Unable to stream an instance after deploying the regular Anbox Cloud using Juju([LP 2110194](https://bugs.launchpad.net/anbox-cloud/+bug/2110194)). This issues exists since the 1.25.0 release. Run the following command to work around this issue:
+
+    ```
+    juju ssh anbox-cloud-dashboard/0 -- "sudo sed -i '/^ASG_API_SERVER_CERTIFICATE: /d' /var/snap/anbox-cloud-dashboard/common/service/config.yaml"
```

Comment:
We need to point our the security implications of this.

Reply:
Does the note below cover this ?

Reply:
Please check the polished note below